mix.exs

```diff
@@ -28,9 +28,15 @@ defmodule VideoRoom.MixProject do
 
   defp deps do
     [
-      {:membrane_rtc_engine, github: "membraneframework/membrane_rtc_engine", override: true},
+      {:membrane_rtc_engine, "~> 0.8.0"},
+      {:membrane_rtp_plugin,
+       github: "membraneframework/membrane_rtp_plugin",
+       branch: "feature/outbound-retransmissions",
+       override: true},
       {:membrane_rtc_engine_timescaledb,
-       github: "membraneframework/membrane_rtc_engine_timescaledb", runtime: false},
+       github: "membraneframework/membrane_rtc_engine_timescaledb",
+       branch: "update-engine",
```

Comment:
You can wait a bit for it to be merged (I've approved the PR).
BTW, do you know why isn't that released?